test/images/windows/powershell-helper/VERSION

```diff
@@ -0,0 +1 @@
+6.2.0
```

Comment:
Should we target 6.2.7
https://github.com/PowerShell/PowerShell/releases/tag/v6.2.7

6.2.0 is from March 2019.

Reply:
> I will defer to someone else for the powershell bits, I'm not fluent there :+)

Other than this the powershell parts look good to me.

Reply:
Done